ComposeSample/dependencies.gradle

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+ext {
+    gradlePluginVersion = "4.2.0-alpha07"
+    kotlinVersion = "1.4.0-rc"
```

Comment:
You can migrate now to stable `1.4.0` if you want: https://kotlinlang.org/docs/reference/whatsnew14.html It was released very recently.